packages/cis_kubernetes_benchmark/kibana/csp_rule_template/csp_rule_template-9129a080-7f48-11ec41308bcdaaf665761478bb6f0d745a5c.json

```
@@ -0,0 +1,26 @@
+{
+    "attributes": {
+        "id": "41308bcdaaf665761478bb6f0d745a5c",
+        "name": "Ensure that the API server pod specification file permissions are set to 644 or more restrictive (Automated)",
+        "tags": [
+            "CIS",
+            "Kubernetes",
+            "CIS 1.1.1",
+            "Master Node Configuration Files"
+        ],
+        "description": "Ensure that the API server pod specification file has permissions of `644` or more restrictive.\n",
+        "rationale": "The API server pod specification file controls various parameters that set the behavior of the API server. You should restrict its file permissions to maintain the integrity of the file. The file should be writable by only the administrators on the system.\n",
+        "default_value": "By default, the `kube-apiserver.yaml` file has permissions of `640`.\n",
+        "impact": "None\n",
+        "remediation": "Run the below command (based on the file location on your system) on the\nmaster node.\nFor example,\n```\nchmod 644 /etc/kubernetes/manifests/kube-apiserver.yaml\n```\n",
+        "benchmark": {
+            "name": "CIS Kubernetes V1.20",
+            "version": "v1.0.0"
+        },
+        "enabled": true,
+        "muted": false,
+        "rego_rule_id": "cis_1_2_2"
+    },
+    "id": "csp_rule_template-41308bcdaaf665761478bb6f0d745a5c",
```

```suggestion
    "id": "csp_rule_template-9129a080-7f48-11ec41308bcdaaf665761478bb6f0d745a5c",
```
or rename the file to
```
packages/cis_kubernetes_benchmark/kibana/csp_rule_template/csp_rule_template-41308bcdaaf665761478bb6f0d745a5c.json
```